scalding-core/src/main/scala/com/twitter/scalding/DescribedExecutionApp.scala

```diff
@@ -0,0 +1,73 @@
+package com.twitter.scalding
+
+object ArgOptions extends Enumeration {
```

Comment:
can we use `sealed abstact class` and `object` subclasses instead? `Enumeration has a lot of issues and many consider it non-idiomatic.